changelog.md

```diff
@@ -4,6 +4,42 @@
 
 // TODO
 
+* Adds a `danger process` command, this command takes amn argument of a process to run which expects the Danger DSL as JSON in STDIN,
+  and will post a DangerResults object to it's STDOUT. This frees up another process to do whatever they want. So, others
+  can make their own Danger runner.
+
+  An example of this is [Danger Swift][danger-swift]. It takes a [JSON][swift-json] file via [STDIN][swift-stdin], [compiles
```

Comment:
Minor clarification: `JSON file` may be more accurate as `JSON document` if I'm understanding correctly.

Reply:
You're correct, I've just updated the changelog: https://github.com/danger/danger-js/commit/d0fc1eb6037feb68508c18e7bc0da0a603a282fb